sdk/core/Azure.Core.TestFramework/src/IgnoreOnNet5Attribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Azure.Core.TestFramework
+{
+    /// <summary>
+    /// Marks that a test should not be executed in NET5 test runs.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = false, Inherited = false)]
```

Comment:
Should be inheritable like other test attributes. Both Core and Key Vault use inheritance a lot.